dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/cluster/ClusterManager.java

```diff
@@ -48,6 +49,10 @@ public class ClusterManager {
     @Autowired
     private RegistryClient registryClient;
 
+    @Autowired
+    @Lazy
```

Comment:
Please remove `@Lazy`, this is not a good practice, we should avoid this.

Reply:
remove @Lazy